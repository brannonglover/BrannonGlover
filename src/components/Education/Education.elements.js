import styled from 'styled-components/macro';

export const EducationWrapper = styled.div`
  padding: 1em .5em .5em;
`;

export const Header = styled.h1`
  margin: .7em 0 .5em;
  font-size: 1.5em;
`;

export const Title = styled.h2`
  font-size: 1em;
  margin: 0;
  padding: 0;
`;

export const School = styled.p`
  margin: 0;
  font-style: italic;
`;

export const Major = styled.div`

`;