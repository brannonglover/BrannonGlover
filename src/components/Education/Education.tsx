import React from 'react';
import { EducationWrapper, Header, Title, School, Major } from './Education.elements';
import data from '../../assets/info.json';

export default function Education() {
  return (
    <>
      <EducationWrapper>
        <Header>Education</Header>
        {data.school.map(school => (
          <React.Fragment key={school.id}>
            <Title>{school.degree}</Title>
            <School>{school.school}</School>
            <Major>{school.major}</Major>
          </React.Fragment>
        ))}
      </EducationWrapper>
    </>
  )
}