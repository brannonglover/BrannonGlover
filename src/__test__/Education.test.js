import React from 'react';
import { render } from '@testing-library/react';
import Education from '../components/Education/Education';

test('Render Education component', () => {
  const { asFragment } = render(<Education />);

  expect(asFragment()).toMatchSnapshot();
});