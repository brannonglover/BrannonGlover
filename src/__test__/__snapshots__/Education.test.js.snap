// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Education component 1`] = `
<DocumentFragment>
  <div
    class="Educationelements__EducationWrapper-rr5tu7-0 hociVB"
  >
    <h1
      class="Educationelements__Header-rr5tu7-1 cOcawK"
    >
      Education
    </h1>
    <h2
      class="Educationelements__Title-rr5tu7-2 cGsNaP"
    >
      Associate, Applied Science
    </h2>
    <p
      class="Educationelements__School-rr5tu7-3 jNqMRH"
    >
      Sandhills Community College
    </p>
    <div
      class="Educationelements__Major-rr5tu7-4"
    >
      Internet Technologies
    </div>
  </div>
</DocumentFragment>
`;
